### DISCUSSION TITLE
OLED screen not working

### DISCUSSION THREAD

**Discussion by frenchriverco** [taoteh1221/Slideshow_Crypto_Ticker#43] [Q&A] [answered]
Hi there,

I'm pretty new to using a Raspberry Pi in general, but I really wanted this project to work.  I'm using a Raspberry Pi Zero 2W and I've managed to get the ticker to work via HDMI on an external monitor.  But, for whatever reason, I'm unable to load anything on my 2.23 inch waveshare OLED, even though I have confirmed that the screen works.

After installing and rebooting, I'm just left with a black screen.  So far, it only works via HDMI.  Am I missing something? Or is it not compatible with a screen of this type or size?  Any help is greatly appreciated!

**Comment by taoteh1221** [comment, marked as answer]
Hello, sorry to hear it's giving you trouble setting it up. The problem is you need a regular LCD screen (like you did over HDMI), because it requires the desktop environment to launch the chromium browser in kiosk mode. A screen like this would do the trick (small LCD screen): https://smile.amazon.com/dp/B07H8ZY89H

**Reply by taoteh1221** [reply]
IMPORTANT UPDATE: Upon closer inspection of the screen I recommended, IT LOOKS LIKE YOU'D HAVE TO SOLDER IT TO THE PI. I do know for the raspi 3 / raspi 4 you can get screen cases THAT PLUGIN TO THE PIE WITHOUT SOLDERING. Hopefully you can find an affordable screen that simply plugs in to your zero 2 w?

**Reply by frenchriverco** [reply]
Thanks for the quick reply.  I've ordered an LCD and hopefully that does the trick!

**Reply by taoteh1221** [reply]
Cool, good luck! :) Holler if you need anymore help. 👍

**Reply by frenchriverco** [reply]
So I got my new LCD 3.5" screen with direct GPIO pins and the application pulled up great on it after running a script to change from HDMI.  However, the fonts were much too large to display on the screen.  I attempted to edit the config file with the recommended font sizes, but now I'm just seeing a black screen with no information.  The screen is still lit up, but either the text is somehow placed off the screen, or it's just not pulling data.  Again, I used the recommended font sizes and screen positions that were indicated in the config file.  Any thoughts?

**Reply by taoteh1221** [reply]
Hmm. Do you have command line access with a terminal app or SSH login? Have you tried running the restart command: ~/ticker-restart

If that doesn't work, maybe try resetting to the default config font sizes, and then adjust the sizes / positions a little bit slower? Then reload after every incremental config change with the same above command, and I think you should be able to slowly dial it in? We'll figure it out, sounds like you almost got it. Also make sure you install the latest version, there are a few blank screen bug fixes in it.

**Reply by frenchriverco** [reply]
I was able to Alt+Tab over to Terminal and do the ~/ticker-restart; same thing, just a black screen.  Is there a way to reset default config, or is the only way to reinstall?

**Reply by frenchriverco** [reply]
I went with a fresh reinstall and did small changes like you recommended.  And it worked! I am so pleased, as this is my first Raspi project.  Thank you very much!  I still want to mess with configuring different assets, but I am very happy with how it's turned out so far!

**Reply by taoteh1221** [reply]
Your welcome, I'm glad you got it working finally! :) Just holler if you have any other questions or feedback, I will do my best to help.
